doc/guide/heom/intro.rst

```diff
@@ -40,4 +40,4 @@ In addition to support for bosonic environments, QuTiP also provides support for
 feriomic environments which is described in :doc:`fermionic`.
 
 Both bosonic and fermionic environments are supported via a single solver,
-:class:`HEOMSolver`, that supports solving for both dynamics and steady-states.
+:class:`~qutip.nonmarkov.heom.HEOMSolver`, that supports solving for both dynamics and steady-states.
```

Comment:
We need to fix this link, but happy for that to happen in a follow up PR unless we have to do it in this one for some reason.